legal-api/src/legal_api/services/filings/validations/incorporation_application.py

```diff
@@ -243,3 +247,49 @@ def validate_incorporation_effective_date(incorporation_json) -> Error:
         return msg
 
     return None
+
+
+def validate_correction_ia(filing: Dict) -> Error:
+    """Validate correction of Incorporation Application."""
+    msg = []
+
+    err = validate_correction_name_request(filing)
```

Comment:
This method has to be called only if there is a nrNumber and the filing being corrected was either a numbered company (no NR) or had a different name request number